### ISSUE TITLE
Needs Game Over screen

### ISSUE THREAD

**Issue by darkf** [darkf/die-yenu-die#2]
If you die, you just sort of tumble over yourself and fall, and nothing else can happen. There should be a game over screen with a "Retry?" option.

**Comment by EFox2413** [comment]
Ive got the gamestate implemented such that it shows up and will disappear after the player presses r or R, however I'm not sure how to go about resetting the game state to default? Would it be better to turn that into a function or to just put all of the code inside of the GameOver gamestate?

**Comment by darkf** [comment]
Yeah there should probably be a `game_init()` or `game_reset()` function (and eventually method). It should reset the map, entities and player.
